docs/source/asr/speech_intent_slot/configs.rst

```diff
@@ -88,6 +88,11 @@ An example config for the model can be:
       subsampling_factor: 4 # must be power of 2
       subsampling_conv_channels: -1 # -1 sets it to d_model
 
+      # Reduction params
+      reduction: null # pooling, subsampling, or null
```

Comment:
What would be the subsampling approach?

Reply:
+1 on explaining the approaches. Suggest using None instead of null

Reply:
The subsampling approach is the initial 4x subsampling that we apply to the encoder. It was there already.
What I added are the reduction parameters which add another 2x subsampling layer in the specified position.

Regarding the null values, across all the configs in NeMo, I have seen the values null specified in yaml files. When we use these files in python, the null values automatically get converted to None type.

Reply:
Looks like the reduction just supports striding. So I suggest to use from ["pooling", "striding"] and raise a "not supported" error if something else is passed.

Reply:
Have added that check in `subsampling.py`, inside the `SubsamplingReductionModule` class